### ISSUE TITLE
[RPP Back Office] Update cost from Purchasing Notes

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#4557]
Prior to the recent updates to the Renewal and Replacement process the RPP Staff was entering permit cost adjustments in the Purchasing Notes. Can we identify the applications with cost adjustments in the Purchasing Notes and update the Permit Cost fields appropriately?

**Comment by moorerst** [comment]
@chrispin Application 2020-1460 is an example - the cost information is in the Payment Notes and the Cost fields have no value. The would have an effect on reporting. Don't know how many applications have this issue.

**Comment by ChrispinP** [comment]
Im going to backlog this since record editing is required. Ideally would like to get reporting done and make sure costs are reporting correctly in their fields before updating any data from a text field.

**Comment by ChrispinP** [comment]
I just realized the addition of a Transaction Amount field will not only allow us to pull more accurate reporting but this field would resolve this issue. Once reporting is in place and field added, I can push Payment Note amounts to the Transaction Amount field.

**Comment by johnclary** [comment]
- Confirm with staff that Transaction Amount should match Total Cost (and is not used to intentionally track the expected cost vs actual paid).  
- Verify that Transaction Amount and Total Cost fields are equal. If not, ask RPP staff on a case by case basis. @ChrispinP please review the data we currently have to see how many discrepancies there are.
- If OK with staff, remove the Transaction Amount field
- Re-evaluate rounding issues with permit costs calculations

**Comment by moorerst** [comment]
Discussed in Sprint Review 3/11 - RPP staff agreed the Transaction Amount field can removed

**Comment by ChrispinP** [comment]
Found 2 Discrepancies in data for Applications 2021-2908(non-matching costs) and 2021-2883(missing costs). Will message Steve next week about these specific records.

**Comment by ChrispinP** [comment]
I have cleaned up the data for Payment Notes and Payment Location. I have also checked tax amounts for all records to make sure they rounded correctly by adjusting the Total Tax field to 3 decimals. I was able to update the Total Cost fields to be correct. There are 5 records with legacy Payment Total Cost amounts of 119.07 instead of 119.08. 

2020-1502, 2020-1497, 2020-1482, 2020-1446, and 2020-1442

All reporting pages have been updated to reflect the updated Costs.
